packages/tupaia-web/src/components/RouterButton.tsx

```diff
@@ -44,7 +45,7 @@ export const RouterLink = ({ to, modal, children, ...props }: RouterButtonProps)
   const link = modal ? { ...location, hash: modal } : to;
 
   return (
-    <Link to={link} {...props}>
+    <Link to={link} {...props} target='_newtab'>
```

Comment:
I probably wouldn't put this here, because this means anywhere that uses `RouterLink` will open everything in another tab